examples/test-site/src/components/Menu/BurgerMenu.token.tsx

```diff
@@ -73,11 +73,7 @@ const $withColumnSubMenuStyles = withDesign({
 });
 
 // Example of custom OverviewLink
-const OverviewLink = asToken(
-  addProps({ children: 'Overview' }),
-)(A);
-
-const $withOverviewLink = withOverviewLink(OverviewLink);
+const $withOverviewLink = withOverviewLink('OOOO');
```

Comment:
I think we should move this example to the demo page and actually override the text